lale/lib/aif360/orbis.py

```diff
@@ -271,19 +107,24 @@ def fit(self, X, y):
         prot_attr_enc = ProtectedAttributesEncoder(
             **fairness_info, remainder="drop", combine="and"
```

Comment:
We can improve this to `combine="keep_separate"`, because with the new mystic-based solution, we no longer need to reduce multiple protected attributes into a single one.

Reply:
Good point, will address!